src/three_by_three_board.py

```diff
@@ -1,7 +1,20 @@
+import src.constants as constants
+
 class ThreeByThreeBoard():
+    # => All of the possible winning combinations
+    WIN_COMBINATIONS = [
+        [0, 1, 2], # Top row
+        [3, 4, 5], # Middle row
+        [6, 7, 8], # Bottom row
+        [0, 3, 6], # First column
+        [1, 4, 7], # Second column
+        [2, 5, 8], # Third Column
+        [0, 4, 8], # Left to Right diagonal
+        [2, 4, 6]  # Right to Left diagonal
+    ]
 
     def __init__(self):
-        self.type = "3x3"
+        self.type = constants.THREE_BY_THREE 
         self.size = 9
```

Comment:
Why not extract the board size out to a constant like a lot of the other data?

Reply:
This board size is meant to be used only in this board, I didnt think it needed to be a global constant. Do you disagree if that's the case?